WaterDock-2.0_Basics.md

```diff
@@ -0,0 +1,76 @@
+**This file is a more detailed version of README.md.**
+N.B. The command line commands in this text file work for Mac but may not work for Windows and Linux. You might need to look up the equivalent commands for these operating systems.
+
+HOW TO SET UP WATERDOCK-2.0 (DETAILED)
+
+1.   Download the zip file or clone using GitHub.

```

Comment:
Can you provide a clone command?

i.e.

```bash
git clone https://github.com/bigginlab/WaterDock-2.0.git
```

Reply:
Good point, added it below the step.